### ISSUE TITLE
Events: Make stats dynamic

### ISSUE THREAD

**Issue by iandunn** [WordPress/wordcamp.org#1180]
The homepage stats were hardcoded for the launch, but we should make them dynamic now that there's time.

https://github.com/WordPress/wordcamp.org/blob/7823c7aa3c076f37e99a9ba8f66050ed569b94e4/public_html/wp-content/themes/wporg-events-2023/patterns/front-cover.php#L33-L49

Some questions:

* Events - Does "this year" mean "this calendar year", or "the past 365 days" ? I think the latter is better.
* Countries - Is the timeframe here the same as events? 
* Participants - Is the timeframe here the same as events? If not, then we should probably add some qualifier like "all time" so that we're not implying that 500k folks have participated this year. If we do want the yearly timeframe, then it may be better to use a stat like `42,600 people joined`, since that is inherently tied to a date range.

For implementation, the # of events and countries can be easily fetched from `wporg_events`. The participants is meetup.com group members + wordcamp attendees. The best way to get those might be to leverage the `wordcamp-reports` plugin. 

All of the counts should be cached in a transient for 24 hours, and then primed via cron every hour. That way users never experience a delay or a stale value.

**Comment by iandunn** [comment]
@dorsvenabili, @fcoveram, @StevenDufresne do you have any thoughts on the questions above?

**Comment by fcoveram** [comment]
> Does "this year" mean "this calendar year", or "the past 365 days" ? I think the latter is better.

Agree. Now I wonder about what's the best copy. `…this year` sounds like "this calendar year" to me, perhaps `…last year` sounds more accurate.

**Comment by iandunn** [comment]
Yeah, `...last year` or `...past year` seem more equivalent to `past 12 months` to me 👍🏻 

@thetinyl, do you have any thoughts?

We could also round off the large numbers, since a few hundred isn't significant, and it'd give us more room to describe what the stat means. e.g., 

* 3.9k events this past year
* 108 countries with groups
* 42k people joined this past year

**Comment by StevenDufresne** [comment]
I think we should be using total numbers, if it's possible, on the top to celebrate the history of the program and introduce social proof and program health indicators closer to where we expect users to do actions.

**Comment by thetinyl** [comment]
> I think we should be using total numbers, if it's possible, on the top to celebrate the history of the program and introduce social proof and program health indicators closer to where we expect users to do actions.

Agree 100% with @StevenDufresne. I think the story is much clearer when we talk about all-time stats vs. year-to-year: WordPress is a thriving global community, always has been, and continues to grow—be a part of it. (.......trying so hard not to make a Broadway pun here....... 🤐)

**Comment by iandunn** [comment]
We should be able to get something close enough to the total that it'd be good enough for this purpose 👍🏻 

One caveat is that Meetup doesn't store historical data for some things; if a group leaves the program then some of the stats are no longer available. I don't think they'll be a significant difference though.
